Eevee.Sleep.Bot/Extensions/DictionaryExtension.cs

```diff
@@ -0,0 +1,9 @@
+namespace Eevee.Sleep.Bot.Extensions;
+
+public static class DictionaryExtensions {
+    public static string GetString<K,V>(this IDictionary<K,V?> dict) {
+        var items = dict.Select(kvp => kvp.ToString());
+        dict.Select(k => k.Value?.ToString()).ToList().ForEach(Console.WriteLine);
```

Comment:
If printing out to console / logging the content is intended, make it a `bool` function argument so it's clear that it logs the content to the console

Also, use the actual loggers instead (see how some `Log` instances are implemented)

Reply:
This is unnecessary.
I forgot to erase the debugging code.